pkg/sbproxy/notifications/handlers/broker_handler_test.go

```diff
@@ -30,6 +30,7 @@ var _ = Describe("Broker Handler", func() {
 	var brokerHandler *handlers.BrokerResourceNotificationsHandler
 
 	var brokerNotification *types.Notification
+	var brokerPlatformCredential *types.BrokerPlatformCredential
```

Comment:
do you plan to add tests cases for failures in credentials activation?

Reply:
currently in case of error in credentials activation i'm just logging it, but not failing the flow.
should i fail it?

Reply:
no (or maybe we should talk about it :)), but you should test it, i.e. a test that fails activation and verifies that flow doesn't break.

Reply:
I actually did put a thought about it and I think we shouldn't fail the flow just because we are unable to activate a valid credential (that were already updated successfully in the platform).
I'm not sure how to test it, onCreate and onUpdate are void functions and activateCredentials is the last action in the method so I can't even check that some other function execution happened.
activateCredentials has the same behavior as putCredentials which has no test for it :)